core/templates/pages/exploration-editor-page/editor-tab/graph-directives/state-graph-visualization.directive.ts

```diff
@@ -222,6 +228,13 @@ angular.module('oppia').directive('stateGraphVisualization', [
             return Math.min(Math.max(value, minValue), maxValue);
           };
 
+          $scope.isCheckpoint = function(nodeId) {
+            var state = ExplorationStatesService.getState(nodeId);
+            if (state) {
+              return state.cardIsCheckpoint;
+            }
```

Comment:
```suggestion
            return !!state && state.cardIsCheckpoint;
```
So that this function always returns a boolean.

Reply:
Done